app/src/controllers/filter/selectors.js

```diff
@@ -68,3 +73,18 @@ export const dirtyFilterIdsSelector = createSelector(
       })
       .map((item) => item.id),
 );
+
+export const filterFromQuerySelector = createSelector(
+  queryParametersSelector,
+  ({ [SIZE_KEY]: size, [SORTING_KEY]: sort, [PAGE_KEY]: page, ...filters }) => {
+    if (isEmptyObject(filters)) {
+      return null;
+    }
+
+    return Object.entries(filters).reduce(
+      (acc, [filterKey, filterValue]) => collectFilterConditions(filterKey, filterValue, acc),
+
+      { conditions: [] },
+    );
+  },
```

Comment:
Can be done using existing utils:

`{ conditions: addFilteringFieldToConditions(collectFilterEntities(filters)) }`

Also please move `collectFilterEntities` to `filter/utils` as this place looks more preferable for it.